pf4j/src/main/java/ro/fortsoft/pf4j/PluginWrapper.java

```diff
@@ -54,7 +54,7 @@ public String getPluginPath() {
 	 * for this plug-in. The class loader can be used to directly access
 	 * plug-in resources and classes.
 	 */
-    public PluginClassLoader getPluginClassLoader() {
+    public ClassLoader getPluginClassLoader() {
```

Comment:
I know that this change makes `PluginWrapper` testable but I think that it's a drawback from the API point of view. If it's not another option and if this change doesn't break the current code (at compilation) I am ok with this modification.